sql/core/src/main/scala/org/apache/spark/sql/execution/joins/BroadcastHashJoinExec.scala

```diff
@@ -44,7 +45,7 @@ case class BroadcastHashJoinExec(
     buildSide: BuildSide,
     condition: Option[Expression],
     left: SparkPlan,
-    right: SparkPlan)
+    right: SparkPlan, isNullAwareAntiJoin: Boolean = false)
```

Comment:
nit: Can you the `isNullAwareAntiJoin` on a new line.

Also, just to make sure: This should show up in the .sql.out files: This should change the string representation of the node and thus you would probably need to update some of the `.out` files. I think this should be the intended behavior.

Reply:
> nit: Can you the `isNullAwareAntiJoin` on a new line.
> 
> Also, just to make sure: This should show up in the .sql.out files: This should change the string representation of the node and thus you would probably need to update some of the `.out` files. I think this should be the intended behavior.

I've check on the sql.out file format, Node string representation is not part of the output.

```
* The format for golden result files look roughly like:
 * {{{
 *   -- some header information
 *
 *   -- !query
 *   select 1, -1
 *   -- !query schema
 *   struct<...schema...>
 *   -- !query output
 *   ... data row 1 ...
 *   ... data row 2 ...
 *   ...
 *
 *   -- !query
 *   ...
 * }}}
```